web/src/engine/websites/MangaLink.ts

```diff
@@ -0,0 +1,21 @@
+import { Tags } from '../Tags';
+import icon from './MangaLink.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Madara from './decorators/WordPressMadara';
+import * as Common from './decorators/Common';
+
```

Comment:
Avoid adding website specific anti-scraping detections into the general `AntiScrapingDetection`.
Instead, place it directly into the related file for the website:
```typescript
AddAntiScrapingDetection(async (dom) => {
    return dom.querySelector('form#lsrecaptcha-form[action*="/.lsrecap/recaptcha?"]') ? FetchRedirection.Interactive : undefined;
});
```

Reply:
Oh, ok,